OpenChange/RTFHandler.m

```diff
@@ -660,22 +660,17 @@ - (void) parseStyleSheet
    } while (count != 0); 
 }
 
-//
-//
-//
-- (void) parsePicture
+- (void) parseIgnoringEverything
 {
+  unsigned int count = 1;
   // Ignore everything. But we cannot parse it blindly because it could have
   // binary data with '}' and '{' bytes, so disasters can happen and they will
-  unsigned int count = 0;
-  const char *cw;
-
   do
     {
       if (*_bytes == '\\')
         {
           unsigned int binary_size, len = 0, cw_len;
-          cw = [self parseControlWord: &len];
+          const char *cw = [self parseControlWord: &len];
```

Comment:
What's the point of this change? I don't see the benefit. I'd rather have the var declaration at top.